### PR TITLE
Patch: Commented unsafe checker

### PR DESCRIPTION
**Pull request by zktony**
## Describe your changes
Patch: Commented unsafe checker

### CODE REVIEW

File: primitives/orderbook/src/recovery.rs
Comment (serhii-temchenko) [inline primitives/orderbook/src/recovery.rs:76]: Please add documentation

File: scripts/check_deviation.sh
Comment (serhii-temchenko) [inline scripts/check_deviation.sh]: Please add `#!/bin/bash` at the beginning

File: pallets/ocex/rpc/src/lib.rs
Comment (serhii-temchenko) [inline pallets/ocex/rpc/src/lib.rs]: Why is this commented out? at least, please add some inline comments why it is commented out and not just removed.

File: pallets/ocex/src/validator.rs
Comment (serhii-temchenko) [inline pallets/ocex/src/validator.rs]: What is this about?

**Review by serhii-temchenko — CHANGES_REQUESTED** [review]

**Review by serhii-temchenko — COMMENTED** [review]
CI should be fixed after merge of PR #869.
Will approve after merge of that PR and re-run and success pass of the CI here.

**Review by serhii-temchenko — APPROVED** [review]

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**
